src/material-experimental/mdc-slider/slider.spec.ts

```diff
@@ -773,6 +773,259 @@ describe('MDC-based MatSlider' , () => {
     });
   });
 
+  describe('slider with change handler', () => {
+    let sliderInstance: MatSlider;
+    let inputInstance: MatSliderThumb;
+    let sliderElement: HTMLElement;
+    let fixture: ComponentFixture<SliderWithChangeHandler>;
+    let testComponent: SliderWithChangeHandler;
+
+    beforeEach(waitForAsync(() => {
+      fixture = createComponent(SliderWithChangeHandler);
+      fixture.detectChanges();
+      testComponent = fixture.debugElement.componentInstance;
+      spyOn(testComponent, 'onChange');
+      spyOn(testComponent, 'onInput');
```

Comment:
It's not a big deal, but it can save you a few lines of code. Instead of having to call `spyOn` on each of these methods, you can declare them as spies instead.

```
onChange = jasmine.createSpy('onChange');
```